src/rtc.c

```diff
@@ -505,6 +505,9 @@ bool RTC_init(hourFormat_t format, rtcMode_t mode, sourceClock_t source, bool re
   HAL_RTCEx_EnableBypassShadow(&RtcHandle);
 #endif
 
+    HAL_NVIC_SetPriority(RTC_Alarm_IRQn, RTC_IRQ_PRIO, RTC_IRQ_SUBPRIO);
+    HAL_NVIC_EnableIRQ(RTC_Alarm_IRQn);
+
```

Comment:
Don't think it is a good idea to add this. Arlarm IRQ should be enable only when alarm is set and enabled.
Which is done before by:
```
      if (isAlarmASet) {
        RTC_StartAlarm(ALARM_A, alarmDay, alarmHours, alarmMinutes, alarmSeconds, alarmSubseconds, alarmPeriod, alarmMask);
      }
#ifdef RTC_ALARM_B
      if (isAlarmBSet) {
        RTC_StartAlarm(ALARM_B, alarmBDay, alarmBHours, alarmBMinutes, alarmBSeconds, alarmBSubseconds, alarmBPeriod, alarmBMask);
      }
#endif
```